README.md

```diff
@@ -93,21 +129,82 @@ Errors that occur during ingest will end up in the dead letter processing queue,
 
 ## Development
 
+Install [NVM](https://github.com/nvm-sh/nvm) to manage your Node.js environment.
+
+```
+# uses version in .nvmrc
+nvm install
+nvm use 
+```
+
+The package-lock.json was built with npm 8.5.0, so use at least this version.
+
+There are several useful npm commands available to use locally:
 
 ```
 # Install dependencies in package.json
-$ npm install
+npm install
 
 # Run the build command in each of the packages (runs webpack)
-$ npm run build
+npm run build
+
+# Run ESLint
+npm run lint
```

Comment:
this command is in a different commit, but will eventually exist in the codebase